app/ldap.go

```diff
@@ -73,7 +73,7 @@ func (a *App) GetAllLdapGroupsPage(page int, perPage int, opts model.LdapGroupSe
 		}
 	} else {
 		ae := model.NewAppError("GetAllLdapGroupsPage", "ent.ldap.app_error", nil, "", http.StatusNotImplemented)
-		mlog.Error(fmt.Sprintf("%v", ae.Error()))
+		mlog.Error("LDAP groups not found for the configired group", mlog.Any(opts))
```

Comment:
This is not the error expected here, actually the error should be something like `mlog.Error("Unable to use ldap", mlog.Err(err))`

Reply:
UPDATED

Reply:
Same here: 

> 
```suggestion
		mlog.Error("Unable to use ldap", mlog.Error(ae))
```